### PR TITLE
added info about fac fee not affecting any transaction limits

### PR DESCRIPTION
**Pull request by kellymore**
Please check to see if it is placed okay

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -5033,9 +5033,9 @@ can-use-dom@^0.1.0:
   integrity sha1-IsxKNKCrxDlQ9CxkEQJKP2NmtFo=
 
 caniuse-lite@^1.0.30000981, caniuse-lite@^1.0.30000989, caniuse-lite@^1.0.30001017, caniuse-lite@^1.0.30001043, caniuse-lite@^1.0.30001061:
-  version "1.0.30001066"
-  resolved "https://registry.yarnpkg.com/caniuse-lite/-/caniuse-lite-1.0.30001066.tgz#0a8a58a10108f2b9bf38e7b65c237b12fd9c5f04"
-  integrity sha512-Gfj/WAastBtfxLws0RCh2sDbTK/8rJuSeZMecrSkNGYxPcv7EzblmDGfWQCFEQcSqYE2BRgQiJh8HOD07N5hIw==
+  version "1.0.30001157"
+  resolved "https://registry.npmjs.org/caniuse-lite/-/caniuse-lite-1.0.30001157.tgz"
+  integrity sha512-gOerH9Wz2IRZ2ZPdMfBvyOi3cjaz4O4dgNwPGzx8EhqAs4+2IL/O+fJsbt+znSigujoZG8bVcIAUM/I/E5K3MA==
```
Comment (ShreyaThapa) [inline yarn.lock:5038]: I wonder what this change is. 🤔

File: pages/concepts/facilitator-fee.mdx [hunk 2]
```diff
@@ -29,6 +29,8 @@ Fees are programmatically set on an individual transfer API request. Within a tr
 
 A fee object is made up of a `_links` and an `amount` JSON object. The `_links` object contains `charge-to`, which represents the associated source or destination [Customer](https://docs.dwolla.com/#customers) or [Account](https://docs.dwolla.com/#accounts) resource that will assume the fee. The `amount` object contains `value` and `currency` keys corresponding to the fee amount and `USD` respectively.
 
+When a facilitator fee is added to a transfer request, it is treated as a separate transfer which does not affect a Customer's weekly, or per transaction limit. 
+
```
Comment (ShreyaThapa) [inline pages/concepts/facilitator-fee.mdx:33]: This looks good. 👍

File: yarn.lock (hunk 1 above)
Comment (kellymore) [reply yarn.lock:5038]: oh poopers. let me see

Comment (kellymore) [reply yarn.lock:5038]: Oh I think it's bc I ran npx browserslist@latest --update-db

https://github.com/browserslist/browserslist/issues/492https://github.com/browserslist/browserslist/issues/492

Comment (kellymore) [reply yarn.lock:5038]: Is it still ok to merge, or should I edit?

Comment (ShreyaThapa) [reply yarn.lock:5038]: Ah, might bee fine then. 😅 I'd like to ask @sausman to weigh in as well when he has a chance!

Comment (sausman) [reply yarn.lock:5038]: @ShreyaThapa +1 should be fine

**Pull request merged**